src/mjolnir/osmway.cc

```diff
@@ -9,7 +9,8 @@ using namespace valhalla::baldr;
 namespace {
 
 constexpr uint32_t kMaxNodesPerWay = 65535;
-constexpr float kMaxOSMSpeed = 255.0f;
+constexpr float kUnlimitedOSMSpeed = 255.0f;
```

Comment:
i wonder if this woudl feel less special if we tied it to the number of bits that empose this. specifically:

    constexpr float kUnlimitedOSMSpeed = std::numeric_limits<uint8_t>::max();

again dont feel strongly about it. just offering it up if it was annoying you

Reply:
Good thought 👍